assets/js/script.js

```diff
@@ -0,0 +1,199 @@
+// Assignment code
+var generateBtn = document.querySelector("#generate");
+
+// G L O B A L  A R R A Y S
+
+// arrays containing characters, symbols & numbers used in password generation
+
+// symbols
+const symbolCharacter = [
+  "!",
+  "#",
+  "$",
+  "%",
+  "&",
+  "'",
+  "(",
+  ")",
+  "*",
+  "+",
+  ",",
+  "-",
+  ".",
+  "/",
+  ":",
+  ";",
+  " < ",
+  "=",
+  " > ",
+  " ? ",
+  "@",
+  "[",
+  "\\",
+  "]",
+  " ^ ",
+  "_",
+  "`",
+  "{",
+  "|",
+  "}",
+  "~",
+];
+// numbers
+const numberCharacter = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"];
+// lowercase
+const lowerCaseCharacter = [
+  "a",
+  "b",
+  "c",
+  "d",
+  "e",
+  "f",
+  "g",
+  "h",
+  "i",
+  "j",
+  "k",
+  "l",
+  "m",
+  "n",
+  "o",
+  "p",
+  "q",
+  "r",
+  "s",
+  "t",
+  "u",
+  "v",
+  "w",
+  "x",
+  "y",
+  "z",
+];
+// uppercase
+const upperCaseCharacter = [
+  "A",
+  "B",
+  "C",
+  "D",
+  "E",
+  "F",
+  "G",
+  "H",
+  "I",
+  "J",
+  "K",
+  "L",
+  "M",
+  "N",
+  "O",
+  "P",
+  "Q",
+  "R",
+  "S",
+  "T",
+  "U",
+  "V",
+  "W",
+  "X",
+  "Y",
+  "Z",
+];
+
+// F U N C T I O N
+// const createPassword = function () {}
+// create password function
+function createPassword() {
+  // request input
+  const passwordLength = parseInt(
+    prompt("Please input a password length between 8 and 128")
+  );
+  // validate input (between 8-128)
+  if (passwordLength < 8 || passwordLength > 128) {
```

Comment:
We need an additional check here to protect against users entering non numeric characters - how can we check this?